src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchRunConfiguration.java

```diff
@@ -100,8 +101,9 @@ public int getSplitMessagesLongerThan() {
     public Map<String, Object> createData() {
         Map<String, Object> data = new LinkedHashMap<>();
         Date date = new Date();
-        data.put(TIMESTAMP, ZonedDateTime.now(ZoneOffset.UTC).format(UTC_MILLIS));
-        data.put(TIMESTAMP_MILLIS, date.getTime());
+        ZonedDateTime zdt = Instant.now().atZone(ZoneId.of("UTC"));
+        data.put(TIMESTAMP, zdt.format(UTC_NANOS));
+        data.put(TIMESTAMP_MILLIS, zdt.toInstant().toEpochMilli());
```

Comment:
Instead of converting from Instant to ZonedDateTime back to Instant:

```suggestion
        Instant now = Instant.now();
        data.put(TIMESTAMP, now.atZone(ZoneId.of("UTC")).format(UTC_NANOS));
        data.put(TIMESTAMP_MILLIS, now.toEpochMilli());
```